### ISSUE TITLE
Dragon Overview stopped rendering properly -- no boxes

### ISSUE THREAD

**Issue by super-aardvark** [eternaldensity/Sandcastle-Builder#1512]
My Dragon Overview pane has no boxes in it.  I have the Dragon Overview boost, and it was working before, but now it's got the axis labels and that's it -- which, come to think of it, is exactly how it looked before I got the Dragon Overview boost..  However, I can click the invisible boxes to go to the specified NP, and the tooltip shows the NP number and any dragons there.

**Comment by eternaldensity** [comment]
When you say it was working 'before', was it working after upgrading to
4.06?

**Comment by super-aardvark** [comment]
I'm not 100% sure.  I don't reload the page very often, and didn't know about the 4.06 update until I went looking for info about this bug.  I reloaded the page today to try to fix the bug, and I don't know when the last time was,  So, I guess if there's not some kind of auto-update feature, then either it broke while still on 4.05, or it was working on 4.06.

I first noticed the bug when I switched the overview from Negative OTC to Positive OTC after spending a bunch of time collecting shards in negative catpix.

Another thing: at some point in the last several days, a new pair of left/right buttons appeared, which switch between "Positive OTC" and "Positive t1i" (and same for Negative).  These have been similarly blank from the moment they first appeared, but since I had (and have) no idea what they're for, I didn't think anything of it.

Let me know if there's anything I can do on the console to get debug info or anything else that might help.

**Comment by super-aardvark** [comment]
Ok, so, I woke up this morning and pressed the ">>" button on the Overview, and all the boxes appeared.

When the problem started, I had been at my most negative NP, mostly idle but using Stealth Cam to collect a shard each ONG.  The Overview was on Negative OTC, and I clicked ">>" and the boxes disappeared.  I also remember, I tried using Now Where Was I, but it didn't take me to positive NP.  Now, I've been at my highest positive NP, doing the same thing (Stealth Cam each ONG), with the Overview on Positive OTC (and no boxes showing), and clicking ">>" made the boxes appear.

So, maybe it has something to do with the highest NP being negative (I know there's a badge for that, which I'd gotten long before reaching The End).
